test/gen_terminal_strings_test.html

```diff
@@ -1,6 +1,6 @@
 <html>
 <body>
-<script src="../../build/spot.js"></script>
+<script src="../build/spot.js"></script>
 
 <script>
```

Comment:
These could be converted to Mocha tests too.
For example:
describe('terminal strings drawn from [a, b, c]', function(){
   it('with minimum length 2, maximum length 2', function(){
       assert(....)
   }
}
and so on. You can assume that the currently generated sets of terminal strings are correct in setting up the expected values.